client/webserver/site/src/html/order.tmpl

```diff
@@ -82,7 +82,7 @@
       </div>
       <div class="order-datum">
         <div>[[[Age]]]</div>
-        <div data-stamp="{{$ord.Stamp}}"></div>
+        <div data-stamp="{{$ord.SubmitTime}}"></div>
```

Comment:
Left as `data-stamp` so it is one of the `stampers` used in order.ts with `setStamp` to covert these fields into ages based on this attribute's presence